graphs/omen-thumbnailmapping/schema.graphql

```diff
@@ -0,0 +1,5 @@
+type OmenThumbnailMapping @entity {
+  id: Bytes! # same as marketAddress
+  marketAddress: Bytes! # address
```

Comment:
Do we need to store both? If ID already suffices, let's keep ID.

Reply:
I wanted to keep both. Event contains `marketAddress` (not `id`), so it's not clear why the graph wouldn't have `marketAddress`, as a 3rd user I would expect it there. But on the other hand, `id` is required by the graph, otherwise it errors out.

Reply:
Your point is valid - but from a data standpoint, we have unnecessary redundancy.
Would it be acceptable to have ID documented as "marketId"?

Reply:
Yeah I know, but usually I prefer clarity. Right now it nicely maps to ImageUpdated event (also, I don't see any docs there where we could document it, figuring out how Omen's graphs work when we joined with Evan was painful 🙈 ) :

![Screenshot by Dropbox Capture](https://github.com/gnosis/labs-contracts/assets/15619339/b53a8c1b-dce9-464c-8b15-e9f90f865ba0)

If we want to be efficient, I can also remove `changer`, as that's not needed outside of the contract.

But do you have some numbers for this, please? I expected that this graph would cost nothing compared to the rest of Omen-related graphs.

If that's true, don't see a problem with keeping it like this, but if there are big cost savings by removing marketAddress+changer, sure let's do that!

Reply:
The cost is associated with syncing time - more entities or fields equal more time to sync.
I accept the argument that this is anyways a lot smaller than the other Omen subgraphs, hence should sync quick enough. If that's not the case, we can revisit this.